src/components/Inputs/Input.jsx

```diff
@@ -1,14 +1,23 @@
 import PropTypes from 'prop-types';
 import '../Tables/tables.scss';
 
-const Input = ({ type, value, placeholder, required, onChange, id }) => (
+const Input = ({
+  type,
+  value,
+  placeholder,
```

Comment:
placeholder = undefined <- to indicate that the default value.  See also:
https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values